Gemfile

```diff
@@ -135,7 +135,7 @@ gem 'strong_migrations'
 gem 'swagger-blocks'
 gem 'typhoeus'
 gem 'utf8-cleaner'
-gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'master'
+gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'space-force'
```

Comment:
Do we want the space-force branch to be merged into master in vets_json_schema and still point to the master branch here?

Reply:
I will be pointing this back to master once the schema is merged.